docs/src/modules/components/Ad.js

```diff
@@ -1,11 +1,12 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Tooltip from '@material-ui/core/Tooltip';
 import Paper from '@material-ui/core/Paper';
-import InfoIcon from '@material-ui/icons/InfoOutlined';
```

Comment:
What do you think about `PriorityHigh` or `HelpOutlined`. Actual text content can be confusing and therefore even more distracting.

If an icon is too distracting we can make it smaller.

Reply:
All these icons looks good to me. My issue was first with the distribution aspect, and then realizing that the `i` was already providing most of the value (we understand what's going on) without the overhead of an icon (import, bundle, runtime).

Could you explain on what you mean by text content can be confusing?

Reply:
It's confusing for screen readers and tbh just looks bad to the point where I originally thought it was an issue with the Ad. I had no idea it actually included informatio

Reply:
What do you think of removing it?

Reply:
> It's confusing for screen readers

It's not focusable how can a screen reader access it?

Reply:
The internet would be completely unusable if screen readers could only read focusable areas. What screen reader do you use for testing that can't read paragraphs?

> What do you think of removing it?

Reply:
> The internet would be completely unusable if screen readers could only read focusable areas. 

Agree, I'm using VoiceOver, I will spend some time to read https://help.apple.com/voiceover/info/guide/10.8/English.lproj/index.html.

> Makes more sense.

Ok, let's do that :)